Automattic/wp-calypso
# Site Profiler Performance: Add the InsightContent component to display the basic description

Fixes https://github.com/Automattic/dotcom-forge/issues/7482
Depends on https://github.com/Automattic/wp-calypso/pull/91211

## Proposed Changes

* Add the InsightContent component
* Display the basic description for each issue

## Why are these changes being made?

To match the proposed layout at https://www.figma.com/design/P7Jjk4nbUTahUk8aHkG6NQ/Performance-Tool?m=dev&node-id=1124-2591

## Testing Instructions

* Go to `/site-profiler/:url`. Ex: `/site-profiler/wordpress.com`
* Go to the Performance section and check if real data is being shown on the list and the style matches the [proposed layout](https://www.figma.com/design/P7Jjk4nbUTahUk8aHkG6NQ/Performance-Tool?m=dev&node-id=1124-2591)

![CleanShot 2024-05-28 at 17 53 15@2x](https://github.com/Automattic/wp-calypso/assets/5039531/120290f4-3fe5-4939-b11d-258e34adfafd)


## Pre-merge Checklist
